### DISCUSSION TITLE
Add `eslint-config-rome`

### DISCUSSION THREAD

**Discussion by nstepien** [rome/tools#3663] [Suggestions]
Just like [`eslint-config-prettier`](https://github.com/prettier/eslint-config-prettier), I think Rome should publish a `eslint-config-rome` package.

What `eslint-config-prettier` does is it disables ESLint rules that are covered by the prettier formatter.
So `eslint-config-rome` could do the same:
- when Rome's formatter is enabled, `eslint-config-rome` would disable equivalent lint rules in ESLint
- when Rome's linter is enabled, `eslint-config-rome` would disable equivalent lint rules in ESLint
  - it should check which Rome rules are enabled, so if a Rome rule is disabled, the equivalent ESLint rule should not be disabled.

This would allow for a smooth transition from ESLint to Rome, and improve ESLint performance as fewer rules would need to run.
The ESLint ecosystem is quite big, so I wouldn't expect Rome to 100% replace it for some years. Happy to be proven wrong though.

For reference:
- https://github.com/prettier/eslint-config-prettier
- https://prettier.io/docs/en/integrating-with-linters.html

**Comment by strager** [comment]
I think this is a good idea.

I considered something similar for quick-lint-js (my linter), but further: ESLint could run quick-lint-js (in addition to ESLint's original rules). That way the user still has all their lint rules in one place (ESLint). https://github.com/quick-lint/quick-lint-js/issues/682

**Comment by MichaReiser** [comment]
@nstepien Have you tried using `eslint-config-prettier`? I would expect it to work out of the box since Rome embraces Prettier's formatting. This isn't saying that we shouldn't publish such a package but could be a short-term workaround.

**Reply by nstepien** [reply]
Yes we're already using it 👍
But it only covers the formatting part, we still have overlapping rules between ESLint and Rome's linter.

**Reply by nstepien** [reply]
I could be wrong, but I think `eslint-config-rome` could itself extend `eslint-config-prettier` to not duplicate their work.
